pkg/storage/chunk/client/ibmcloud/cos_object_client_test.go

```diff
@@ -518,3 +529,74 @@ func Test_APIKeyAuth(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, resp, strings.Trim(string(data), "\n"))
 }
+
+func Test_TrustedProfileAuth(t *testing.T) {
+	testToken := "test"
+	tokenType := "Bearer"
+	resp := "testGet"
+
+	cosSvr := cosServer(testToken, tokenType, resp)
```

Comment:
could we rename the srv to clearly mention that this is nothing but a fake server?